test/parallel/test-promises-warning-on-unhandled-rejection.js

```diff
@@ -0,0 +1,31 @@
+// Flags: --no-warnings
```

Comment:
That's not how emitWarning works. By default it prints to stderr *and* emits the `'warning'` event. The `--no-warnings` switch suppresses the default stderr output but maintains the `'warning'` event. There is no bug.